ansible/playbooks/nagios/vars_setup_server.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
the playbook `play_setup_server.yml `includes this `vars_serup_server.ym`l and include role Nagios_Server which has a `defaults`.
these two files seem duplicated. 
do we really need this file?